hermes-console-vue/src/components/environment-badge/EnviromentBadge.vue

```diff
@@ -0,0 +1,22 @@
+<script setup lang="ts">
+  interface HermesEnvironmentProps {
```

Comment:
We can also introduce new package named "types" or "@/types" in this component folder